internal/testrunner/runners/system/tester.go

```diff
@@ -2065,6 +2041,25 @@ func (r *tester) checkTransforms(ctx context.Context, config *testConfig, pkgMan
 	if err != nil {
 		return fmt.Errorf("loading transforms for package failed (root: %s): %w", r.packageRootPath, err)
 	}
+
+	// Before stack version 8.14.0, there are some issues generating the corresponding
+	// mappings for the fields defined in the transforms
+	// Forced to not use mappings to validate transforms before 8.14.0
+	// Related issue: https://github.com/elastic/kibana/issues/175331
```

Comment:
Found some issues with the generated mappings for the transforms defined in some packages. If two group fields were defined (with different fields under it), they were not merged. This happened for instance in `github` package.

It looks like it was solved here https://github.com/elastic/kibana/pull/177608 and it is available/solved starting in stack version 8.14.0

Added for now a way to not run validated based on mappings (just in transforms).